Vagrantfile

```diff
@@ -127,7 +127,7 @@ Vagrant.configure("2") do |config|
 
     # Clean test machine:
     config.vm.define "clean", autostart: false do |clean|
-        config.vm.box = "ubuntu/trusty64"
+        config.vm.box = "ubuntu/bionic64"
```

Comment:
Please update the box used by *libvirt* below too.

Reply:
Also in the other two places specifying this box -- maybe it's a good time to define a variable/constant for those two boxes?

Reply:
I can't test it, so maybe you or someone else can do it and test?

Reply:
> Also in the other two places specifying this box -- maybe it's a good time to define a variable/constant for those two boxes?

Maybe, but I don't know ruby, so I tend to just make the "dumb" fix, rather than doing things in more elegant ways.